scipy/optimize/_minimize.py

```diff
@@ -729,6 +740,11 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
         if "hess_inv" in res:
             res.hess_inv = None  # unknown
 
+    if getattr(callback, 'stop_iteration', False):
+        res.success = False
+        res.status = 5
```

Comment:
Why did you choose 5? Any number if fine with me, what matters is that we document this. Ideally we could have an enum or other mapping (code, msg) that we would use for all methods. This would ensure we are consistent.

Reply:
Please see the top post. It was an open question.